app/src/gui/fields/TakePhoto.tsx

```diff
@@ -287,49 +300,57 @@ export class TakePhoto extends React.Component<
     // But it doesn't look like we support masonry right now.
     //
     // It also looks like we don't have multiple photos being returned...
+
+    // We have two properties available here - both are optional
+    const title = this.props.label;
+    const helperText =
+      this.props.helpertext ?? this.props.helperText ?? undefined;
+
     return (
-      <div>
-        {this.props.helpertext || this.props.helperText}
-        <Button
-          variant="outlined"
-          color={'primary'}
-          style={{marginRight: '10px'}}
-          fullWidth={true}
-          onClick={async () => {
-            await this.takePhoto();
-          }}
-        >
-          {this.props.label !== undefined && this.props.label !== ''
-            ? this.props.label
-            : 'Take Photo'}
-        </Button>
-        <FAIMSImageList
-          images={this.state.images}
-          setopen={(path: string) =>
-            this.setState({open: true, photopath: path})
-          }
-          setimage={(newfiles: Array<any>) => {
-            this.props.form.setFieldValue(
-              this.props.field.name,
-              newfiles,
-              true
-            );
-          }}
-          disabled={this.props.disabled ?? false}
-          fieldName={this.props.field.name}
-        />
-        <Typography variant="caption" color="textSecondary">
-          {error_text}{' '}
-        </Typography>
-        <FaimsAttachmentManagerDialog
-          project_id={this.props.form.values['_project_id']}
-          open={this.state.open}
-          setopen={() => this.setState({open: false})}
-          filedId={this.props.id}
-          path={this.state.photopath}
-          isSyncing={this.props.issyncing}
-        />
-      </div>
+      <MediaQueryWrapper>
+        {isMobile => (
+          <div>
+            {title && <h3>{title}</h3>}
+            {helperText && <p>{helperText}</p>}
+            <Button
+              variant="contained"
+              color="primary"
+              fullWidth={isMobile ? true : false}
+              onClick={this.takePhoto}
+            >
+              Take photo
+              <span style={{width: 10}} />
+              <CameraAltIcon />
+            </Button>
```

Comment:
This is the only part that actually changed - FYI

Reply:
Happy with the layout change.